README.md

```diff
@@ -237,3 +252,300 @@ We rely on many tools and services for which we are grateful ❤ to their develo
 |    <a href="https://saucelabs.com"><img src="./.github/PoweredBySauceLabs.svg" width="150" /></a>    | Cross-browser testing platform       |
 | <a href="https://webdriver.io/"><img src="https://webdriver.io/img/webdriverio.png" width="50"/></a> | WebDriver test framework for Node.js |
 |        <a href="https://www.argos-ci.com/"><img src="./.github/ArgosCI.svg" width="150"/></a>        | Visual regression system             |
+
+## Utilities API
+
+Several utility functions are provided in the [`util`](./src/util) module for traversing and manipulating the DOM. These may be useful for theme and extension authors when there is a need to modify the HTML structure of the document (e.g. adding additional purely presentational elements such as social sharing buttons). Caution should be taken to not overly modify the content. Only use these functions for things that are not possible with CSS alone.
```

Comment:
I know we discussed this previously and decided to continue with this use case for now, but I feel uncomfortable explicitly encouraging using these utilities for things like adding
> (e.g. adding additional purely presentational elements such as social sharing buttons)

I would at least want us to have a mechanism for allowing user configuration of these elements, as otherwise the themes lose reusability across users, not to mention the performance/theme size/accessibility hits it incurs.

I think these will be better suited if we thought of them as template extensions for Encoda.
A [full fledged templating support](https://github.com/stencila/encoda/issues/389) will probably be best, but if we still want we can consume Thema JS "template extensions" during Encoda's encoding, applying any transformations at compile time, similar to a Server Side Rendering approach.

Either way, I just wanted to make a note of this, but will merge and we can revisit once we're [proofing/editing the README](https://github.com/orgs/stencila/projects/4#card-33640131).